### PR TITLE
JBDS-4026 Bundled binaries sha256 is not verified after download

### PR DESCRIPTION
**Pull request by dgolovin**

### CODE REVIEW

**Comment by dgolovin** [comment]
test this

File: gulpfile.js [hunk 1]
```diff
@@ -151,23 +151,49 @@ gulp.task('create-7zip-archive', function(cb) {
...
+          reqs['jbds.jar'].sha256sum = body;
+          resolve();
+        });
       }
-    }
-  });
+    });
+  };
+
+  return Promise.resolve()
```
Comment (jrichter1) [inline gulpfile.js]: Is this line really necessary?

Comment (dgolovin) [inline gulpfile.js]: This line triggers Promises chain. It just looks better than just calling first function to start the chain.

Comment (jrichter1) [inline gulpfile.js]: Well, if you look at the model classes, the promise chains usually start with the first business function. I'm not a fan of this padded construction (but it's no big deal either).

**Pull request merged**